recipe/meta.yaml

```
@@ -21,6 +22,7 @@ requirements:
     - poetry-core
     - pip
   run:
+    - httpx >=0.23.0,<1
     - python
     - pydantic >=1.0.0,<3.0.0
```

Given that the `.toml` file specifies 2 compatible but different constraints depending on the exact python version, I would err on the side of caution and use the intersection of constraints. What do you think ?
```suggestion
    - pydantic >=2.7.4,<3.0.0
```

---

I went with selectors to reflect source 1:1

---

I'm not sure that selectors are precise enough to match the source (3.12.**4** cutoff) ?

---

The original issue was https://github.com/pydantic/pydantic/issues/9607 and I think it's fine to not include all py312 for pydantic 2

---

Looking at it again, I see that using the specifiers is stricter than the exact cutoff used in the source. I agree this should work.